lib/app/components/component_map.dart

```diff
@@ -15,21 +16,32 @@ class UiElement {
 
 abstract class Elements {
   static const String BUTTONS = "Buttons";
+  static const String CHECKBOXES = "Checkboxes";
 
   static List<UiElement> ComponentList = [
     const UiElement(name: BUTTONS, componentList: Buttons()),
-    UiElement(name: "lightGreenAccent", componentList: Container(color: Colors.lightGreenAccent)),
-    UiElement(name: "lightGreen", componentList: Container(color: Colors.lightGreen)),
-    UiElement(name: "amberAccent", componentList: Container(color: Colors.amberAccent)),
-    UiElement(name: "amberAccent", componentList: Container(color: Colors.amber)),
-    UiElement(name: "redAccent", componentList: Container(color: Colors.redAccent)),
+    const UiElement(name: CHECKBOXES, componentList: Checkboxes()),
+    UiElement(
+        name: "lightGreenAccent",
+        componentList: Container(color: Colors.lightGreenAccent)),
+    UiElement(
+        name: "lightGreen", componentList: Container(color: Colors.lightGreen)),
+    UiElement(
+        name: "amberAccent",
+        componentList: Container(color: Colors.amberAccent)),
+    UiElement(
+        name: "amberAccent", componentList: Container(color: Colors.amber)),
+    UiElement(
+        name: "redAccent", componentList: Container(color: Colors.redAccent)),
     UiElement(name: "red", componentList: Container(color: Colors.red)),
-    UiElement(name: "indigoAccent", componentList: Container(color: Colors.indigoAccent)),
+    UiElement(
+        name: "indigoAccent",
+        componentList: Container(color: Colors.indigoAccent)),
     UiElement(name: "indigo", componentList: Container(color: Colors.indigo)),
-    UiElement(name: "orangeAccent", componentList: Container(color: Colors.orangeAccent)),
+    UiElement(
+        name: "orangeAccent",
+        componentList: Container(color: Colors.orangeAccent)),
```

Comment:
It seems unnecessary to create the UI elements again we can just add chechbox

Reply:
I think it's because of reformatting the code. The UI elements are still the same. Do I need to do anything about here?

Reply:
lets don't change convention here so convert it back as it was, only your change should be present.